### DISCUSSION TITLE
Can I improve the quality of generated images with sharpness?

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#2766] [Q&A] [answered]
I would like to know how to improve the quality of the output image? 
Thank you.

**Comment by poor7** [comment]
You have already created several similar posts, but I still don’t understand what you want? Please provide examples of images and their metadata that you receive and examples of what you would like to receive.

**Comment by DavidDragonsage** [comment, marked as answer]
*** Can Quality Be Improved with Sharpness? ***

The answer depends on how you define quality, which is not only a question of your general preferences but also of how you wish the specific subject to be portrayed.

Sharpness will add detail to an image, but in doing so it may introduce an unwanted level of harshness.

The example images use the Fooocus Default preset, the preset that takes effect when use start Fooocus with run.bat. As the captions show, the first image uses a Sharpness of 2 and the second a Sharpness of 8. All other parameters are identical, including the Seed.

I have noticed that the default vertical alignment of these images in this forum makes it more difficult to compare them. I suggest you download them so you can place them side by side - the differences will become very obvious.

The Default preset uses a Sharpness of 2, which produces somewhat softer, less detailed images. This preset also uses the juggernautXL_v8Rundiffusion.safetensors base model. The Juggernaut series is an excellent choice for making portrait quality images. Portraits are very popular in AI imagery, so this is a very appropriate choice for a default model.

Portraits may be considered more flattering when facial details are somewhat less pronounced, particularly with older subjects (like me).

The examples use the prompt: "a field of wildflowers. Trees and a beautiful Afro-Canadian woman in the background". 

The subject was chosen to create a portrait in which the rendering of the Afro hair style gives a very obvious indication of image detail.

But in addition you can see that the yellow fabric has substantially more texture and more obvious folds, so the higher sharpness value could be considered to give the image more authenticity and depth.

More subtly, the skin texture is more detailed and the earring definition clearer. The field and forest also show a little more detail.

But we return to the question, how do you define quality and what do you seek to convey through the particular image?

![Sharpness 2](https://github.com/lllyasviel/Fooocus/assets/162500231/e0bb8d6d-1244-4f5a-8a2c-69884f995b05)
![Sharpness 8](https://github.com/lllyasviel/Fooocus/assets/162500231/e8b3377f-d5b2-4a0d-bfd1-11692e0f6b17)

**Comment by wisdombox44** [comment]
Try to to experiment with different presets when you upscale 2X, some times less steps give you better results. Upscale with lightning preset, then speed, and quality. Compare the results. Lightning some times gives more details. Down Side of lightning is you wil get more deformef images. Experimen with FreeU also this will give more detail and saturation. Every image is different scenerio there is no magic pill.
